src/config.cc

```diff
@@ -1113,13 +1109,13 @@ Class load()
   c.lastMainGroupId  = root.namedItem( "lastMainGroupId" ).toElement().text().toUInt();
   c.lastPopupGroupId = root.namedItem( "lastPopupGroupId" ).toElement().text().toUInt();
 
-  QDomNode popupWindowState = root.namedItem( "popupWindowState" );
+  QDomNode popupWindowState = root.namedItem( "popupWindowState2" );
 
   if ( !popupWindowState.isNull() ) {
     c.popupWindowState = QByteArray::fromBase64( popupWindowState.toElement().text().toLatin1() );
   }
 
-  QDomNode popupWindowGeometry = root.namedItem( "popupWindowGeometry" );
+  QDomNode popupWindowGeometry = root.namedItem( "popupWindowGeometry2" );
```

Comment:
name changed. Is this correct? @shenlebantongying

Reply:
Yes, because the whole popup is changed. Hence need a reset.

Reply:
I think qt should check the state of the windows, not verified.
This should be a common case in qt windows state.

I'll just keep it at the time.